sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestManager.java

```diff
@@ -81,6 +84,11 @@
     private static final ClosedChannelException ON_DEREGISTER =
         ThrowableUtil.unknownStackTrace(new ClosedChannelException(), RntbdRequestManager.class, "deregister");
 
+    private static final EventExecutor requestExpirator = new DefaultEventExecutor(new RntbdThreadFactory(
```

Comment:
New requestManager is created per channel. #ThreadFactorys ~ #channels. 
Is this intentional?

Reply:
Not sure I understand the question - all new event loops are static - so independent of instance count, correct?

Reply:
Its static please ignore it.